core/client/src/main/java/alluxio/client/file/options/ListStatusOptions.java

```diff
@@ -13,6 +13,7 @@
 
 import alluxio.annotation.PublicApi;
 import alluxio.thrift.ListStatusTOptions;
+import alluxio.thrift.LoadMetadataType;
```

Comment:
the thrift type should not be exposed to the application layer, which it currently is ... we need separate classes for Alluxio LoadMetadataType and Thrift LoadMetadataType